LEGO1/viewmanager/viewlodlist.h

```diff
@@ -44,7 +44,7 @@ class ViewLODList : public LODList<ViewLOD> {
 // ??? for now, until we have symbol management
 typedef const char* ROIName;
 struct ROINameComparator {
-	bool operator()(const ROIName& rName1, const ROIName& rName2) const
+	int operator()(const ROIName& rName1, const ROIName& rName2) const
```

Comment:
They did use `bool` in the 1996 source so we should leave it as is, or is there a good reason not to? (other than the compiler warning)

Reply:
It's a compiler error for me, not just a warning, and I couldn't find a good header to include to get bool.

The original header includes some non-standard `std.h` which presumably defined bool. What do you think I should do?

Reply:
I'm assuming you are not using MSVC 4.20? It only emits a warning about the reserved keyword `bool`, but otherwise compiles fine, presumably treating it as `int`. Or are you using different compiler settings?

Either way, depending on the case, we could add a macro `COMPAT_BOOL` that expands to different things based on the compiler used, similar to `COMPAT_CONST`

Reply:
Ah, sorry, I didn't notice that it was only a warning since I was dealing with another actual error too at the time.

I don't think we should have warnings we can avoid but that doesn't need to be part of this PR, I'll take out the change.